util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -1376,6 +1378,24 @@ static bool GetConfig(const Span<const uint8_t> args[],
         "curve": ["ED-25519"],
         "pure": true,
         "preHash": false
+      },)"
+      R"({
+        "algorithm": "ML-DSA",
+        "mode": "keyGen",
+        "revision": "FIPS204",
+        "parameterSets": ["ML-DSA-44", "ML-DSA-65", "ML-DSA-87"]
+      },{
+        "algorithm": "ML-DSA",
+        "mode": "sigGen",
+        "revision": "FIPS204",
+        "parameterSets": ["ML-DSA-44", "ML-DSA-65", "ML-DSA-87"],
+        "deterministic": [false],
+        "messageLength": [{"min": 8, "max": 65536", "increment": 8}]
```

Comment:
```suggestion
        "messageLength": [{"min": 8, "max": 65536, "increment": 8}]
```

Reply:
Thank you, good spot -- that's a bug in the standard too, so I'll let the working group know!